src/fix_hmc.cpp

```diff
@@ -225,11 +225,11 @@ void store_peratom_member(LAMMPS_NS::Atom::PerAtom &stored_peratom_member,
 
 
 template <typename T>
-void restore_peratom_member(LAMMPS_NS::Atom::PerAtom stored_peratom_member,
+void FixHMC::restore_peratom_member(LAMMPS_NS::Atom::PerAtom stored_peratom_member,
                           LAMMPS_NS::Atom::PerAtom &current_peratom_member, int nlocal)
 {
   if (stored_peratom_member.name.compare(current_peratom_member.name)) {
-    printf("NONONONONONONONONONO\n");    //error->all(FLERR, "fix hmc tried to store incorrect peratom data");
+    error->all(FLERR, "fix hmc tried to store incorrect peratom data");
```

Comment:
Same here, the error statement could be more informative.

```suggestion
    error->all(FLERR, "fix hmc tried to store incorrect peratom data. Ensure the input atoms match those used in fix hmc");
```